### ISSUE TITLE
transform/xform  returns 403

### ISSUE THREAD

**Issue by MartijnR** [OpenClinica/enketo-express-oc#565]
"POST /transform/xform/223843e80d30962135a57d0233c94b61?ecid=61a62de6586e9126856bbe3ba9898ffdd9b0c7d83f9834e6209a870a8342bf5c HTTP/1.1" 403
[04/May/2022:16:28:10 +0000]

**Comment by MartijnR** [comment]
find out what can cause a 403 to be returned

**Comment by MartijnR** [comment]
@svadla-oc, a 403 seems to be return for either:
1. exceeding form quota (I don't think that's it)
2. no active account

For the 2nd option, maybe it's [this custom API](https://app.swaggerhub.com/apis/Enketo/enketo-express-oc-account-manager/1.0.0)? There might be no 'account' created for that OC server/study.

You can completely bypass this account manager (and I'm actually not sure if you're normally doing that), with the configuration for:

```
"linked form and data server": {
        "name": "KoBoCAT",
        "server url": "", // SET TO EMPTY OR TO THE DOMAIN OF THE OC SERVER
```

Does that lead to a solution? 

Oddly, I think in all these cases the Enketo API (oc/api/v1) should return a 403 as well but it seems that in your case the issue only occurs during form loading after the API has returned a response.

**Comment by svadla-oc** [comment]
Thanks @MartijnR. We were able to resolve this issue today. 
This OC server was working fine until it was migrated to a different cloud environment. It seems like when the migration happened, some of the calls (e.g. `/OpenClinica/rest2/openrosa/`) from Enketo to OC server were being blocked at the network level which resulted in Enketo throwing that 403 error. Once we allowed the calls from Enketo to go through, it resolved the issue.
